README.md

```diff
@@ -0,0 +1,6 @@
+# Add kinect udev rules for device access
+Download the udev rules files corresponding to your device(s) and place them in `/etc/udev/rules.d/` then restart.
```

Comment:
no user will see this.

Reply:
I don’t know how else to get around this. AFAIK, flatpak doesn’t support adding udev rules within the sandbox.

Reply:
I might have an idea where it can be more visible, but all of this isn't needed actually. The chances are very slim that anyone would find the plugin on flathub before finding it in upstream repo or obs website. Second, even if they do find it, they might try to go to upstream or flatpak manifest repo to file bug report and then... udev instructions will greet them!

Reply:
Ideally, OpenKinect developers would integrate these udev rules in [systemd's hwdb](https://github.com/systemd/systemd/tree/main/hwdb.d), so that they can be used without any custom udev rules. Not a blocker for this plugin to land, of course, but it'd be appreciated if we can coordinate that with OpenKinect developers. Happy to help with that, if you open an issue or get in touch with them, feel free to ping me too.